### ISSUE TITLE
Missing alt attribute in image tag & code refactoring

### ISSUE THREAD

**Issue by prajwalkulkarni** [fineanmol/Hacktoberfest2022#1591]
The images of contributors rendered are missing `alt` attribute, which hinders the a11y. An `alt` attribute with the value corresponding to the user's github username can be added.
 The `contributors.html` has grown very much mainly because of the `contributors` array being declared in the same script tag. This data collection can be modularised outside of the script, which can help in code maintenance and also store the data efficiently.

I'd like to take up this issue and make the above-suggested fixes.

**Comment by ImArnoldDCODES** [comment]
Hey man should we work on this together?
